src/web/event-notifications/event-notification-types/SlackNotificationSummary.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import CommonNotificationSummary from 'components/event-notifications/event-notification-types/CommonNotificationSummary';
+
+class SlackNotificationSummary extends React.Component {
+    static propTypes = {
+      type: PropTypes.string.isRequired,
+      notification: PropTypes.object,
+      definitionNotification: PropTypes.object.isRequired,
```

Comment:
Can we use `shape` here?

Reply:
i'm not familiar with shape .I will explore shape.